catalog/cataloger_diff.go

```diff
@@ -194,56 +487,37 @@ func (c *cataloger) diffFromChild(ctx context.Context, tx db.Tx, childID, parent
 		PlaceholderFormat(sq.Dollar).
```

Comment:
The canonical way to use Squirrel is to create a [single global variable `psql`](https://github.com/Masterminds/squirrel/blame/master/README.md#L77) (sorry for linking to a blame, dunno how to link to a line inside a Markdown file on GitHub).

Reply:
no, this is great - updated the code.